soroban-sdk-macros/src/derive_enum_int.rs

```diff
@@ -20,7 +20,7 @@ fn derives_copy(attrs: &[Attribute]) -> bool {
                 if let Some(_tt) = ml.tokens.clone().into_iter().find(|tt| {
                     // match this token with what we want
                     if let proc_macro2::TokenTree::Ident(id) = tt {
-                        id.to_string() == "Copy"
+                        id == "Copy"
```

Comment:
The value is already inherently comparable and doesn't need converting into a String.